### DISCUSSION TITLE
i am not able to view or download jpg or png images on github.

### DISCUSSION THREAD

**Discussion by AmitShashi** [community/community#56585] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i am not able to view or download jpg or png images on github.

**Comment by RamtinKosari** [comment]
Hi Dear @AmitShashi 
Check each of these steps that may help to fix your problem 👍🏼 :

Clear your browser cache: Sometimes, cached data can cause issues with image loading. Clearing your browser cache can help resolve this.

Check your internet connection: Ensure that you have a stable and reliable internet connection. A slow or intermittent connection may prevent images from loading properly.

Try a different browser: If you're using a specific browser, try accessing GitHub on a different browser to see if the issue persists. This can help determine if the problem is specific to the browser you're using.

Disable browser extensions: Certain browser extensions or plugins may interfere with image loading. Temporarily disable any extensions or plugins you have installed and check if the images load correctly.

Enable JavaScript: Make sure JavaScript is enabled in your browser settings. GitHub uses JavaScript for various functionalities, including image rendering.

Check repository permissions: Ensure that you have the necessary permissions to view and access the repository. If the repository is private, you need to be authenticated and authorized before you can view or download images.

Report the issue: If none of the above steps resolve the problem, consider reporting the issue to GitHub support. They can investigate further and provide assistance specific to your case.

**Comment by sandroamiridze** [comment]
If you are unable to view or download JPG or PNG images on GitHub, there are a few potential reasons and solutions you can try:

    Check the File Path: Make sure the image file you want to view or download is present in the repository and has the correct file extension (.jpg or .png).

    File Permissions: Ensure that the image files are not set to private or restricted access. If they are part of a private repository, you need to have the necessary permissions to access them.

    Browser Extensions: Some browser extensions might interfere with GitHub's functionality. Try disabling any extensions related to privacy, security, or ad-blocking and see if that resolves the issue.

    Cache and Cookies: Clear your browser's cache and cookies, then try accessing the images again. Sometimes, old cache data can cause display issues.

    Temporary GitHub Issue: GitHub can experience temporary issues or outages. Check GitHub's status page or social media channels to see if there are any known problems.

    Firewall or Network Restrictions: If you are accessing GitHub from a work, school, or public network, there may be restrictions on certain file types like images. Try accessing GitHub from a different network or using a VPN.

    Browser Compatibility: Check if you are using a supported and updated browser. Sometimes, using an outdated browser version can lead to display problems.

    Try a Different Browser: If the issue persists, try accessing GitHub and the images through a different browser to see if it's a browser-specific problem.

    GitHub Account Status: Ensure that your GitHub account is in good standing and has not been suspended or limited due to any violations.

If none of these solutions work, you can consider reaching out to GitHub support or checking GitHub community forums to see if others have experienced similar issues and found solutions.

**Comment by sayandcode** [comment]
If your ISP is Jio (cause your name sounds Indian), it could be because Jio blocks the raw github usercontent urls. Check out this [stack overflow answer](https://stackoverflow.com/questions/76292834/jio-blocked-raw-githubusercontent-com#:~:text=Yes%2C%20Jio%20has%20a%20past,URLs%20for%20no%20good%20reason.&text=Reboot%20the%20system%20after%20adding,system%20and%20it%20should%20work.) or [this github issue](https://github.com/orgs/community/discussions/42655) for more details

**Reply by S-G-001** [reply]
Thanks @sayandcode you saved my day.

**Comment by appatalks** [comment]
> i am not able to view or download jpg or png images on github.

:wave: @AmitShashi 

Thank you for reaching out in our community forums! We value your engagement.

A good test I like to do when an issue like this pops up is a quick [Inspect Network](https://developer.chrome.com/docs/devtools/network/) of the assets that are attempting to load. Often times if something fails to load, the networking tab will have the asset show up in red with a ```Status``` code to help identify the root cause of the failed asset which can then be used for further troubleshooting or finding resolution.

![Inspect Network Example](https://github.com/community/community/assets/4163156/137c90ad-4dca-4958-8e97-604d566ce769)

The above posted suggestions by other community members are really great too :+1:  

Warm regards,
AppaTalks
